crates/turborepo-lib/src/shim.rs

```diff
@@ -176,50 +177,237 @@ struct PackageJson {
     version: String,
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize)]
-pub struct LocalTurboState {
-    bin_path: PathBuf,
-    version: String,
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct YarnRc {
+    pnp_unplugged_folder: PathBuf,
 }
 
-impl LocalTurboState {
-    pub fn infer(repo_root: &Path) -> Option<Self> {
-        let local_turbo_path = repo_root.join("node_modules").join(".bin").join({
+impl Default for YarnRc {
+    fn default() -> Self {
+        Self {
+            pnp_unplugged_folder: PathBuf::from("").join(".yarn").join("unplugged"),
```

Comment:
Is this different from `PathBuf::from(".yarn").join("unplugged")`

Reply:
Nope, but let me fix it in the followup PR: https://github.com/nathanhammond/turbo/pull/8/files#diff-22c6f95f72b511609de1411df92b2a92dd25eb5e7847fce1cafb4243baaea4d5

Reply:
nevermind, fixed her.